src/e2e/kotlin/fi/aalto/cs/apluscourses/e2e/fixtures/CommonFixtures.kt

```diff
@@ -71,6 +76,8 @@ class DialogFixture(remoteRobot: RemoteRobot, remoteComponent: RemoteComponent)
     : CommonContainerFixture(remoteRobot, remoteComponent) {
   fun ContainerFixture.sidePanel() = find(ContainerFixture::class.java,
       byXpath("//div[@class='SidePanel']"))
+  fun jPasswordField() = find(JTextFieldFixture::class.java,
```

Comment:
this is something I'm not sure about... `jPasswordField` feels being a very special **Swing** component, whereas others, e.g. `sidePanel` in this same `Fixture` and elsewhere are a bit more abstract. 

IMO, perhaps we could aim for some higher-level abstraction and the person writing the code should not explicitly know, that s/he's working with `jPasswordField` but rather a  more generic `passwordField`. This is mostly about the naming, the code is complete. However, I believe we could/should agree on a common approach here. 

This also concerns L47 (`aPlusStripeButton`).